WonSeungHyun/db/migrations/20221124051443_create_users_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+  CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    email VARCHAR(200) NOT NULL,
+    profile_image VARCHAR(1000) NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+  );
+
+-- migrate:down
+
+DROP TABLE users;
+
```

Comment:
의미 없는 줄 바꿈은 지양해 주세요 !
```suggestion
-- migrate:down
DROP TABLE users;
```